app/controllers/institutions_controller.rb

```diff
@@ -32,11 +31,18 @@ def new
   end
 
   def create
-    if PendingInstitutionInvite.where(invited_user_id: current_user,  status: 'accepted').count > 0
-      no_data_allowed
-      return
+    inst_params = institution_params
+
+    if inst_params.has_key?(:pending_institution_invite_id)
```

Comment:
We should extract the "get param" logic so we can write:
```
id = get_pending_invite_id
```
Then `id` could be `nil`